sklearn/covariance/_elliptic_envelope.py

```diff
@@ -160,7 +160,7 @@ def fit(self, X, y=None):
 
         Parameters
         ----------
-        X : {array-like, sparse matrix} of shape (n_samples, n_features)
+        X : {array-like} of shape (n_samples, n_features)
```

Comment:
No need for the brackets when there's only a single option
```suggestion
        X : array-like of shape (n_samples, n_features)
```

Reply:
Thank you @jeremiedbb, I have updated the commit based on your suggestion.